### DISCUSSION TITLE
Boost button

### DISCUSSION THREAD

**Discussion by KoXz666** [Rem0o/FanControl.Releases#2454] [Ideas]
Hi, I have perfectly balanced fans almost all of the year, but when is really hot outside, I need to push boost button and make everything spin faster. I would be glad to do that without manual control or making different profiles and chaos

**Comment by ionGL** [comment]
Do you have a profile that you are satisfied with? Just click the three dots on the top right and save configuration as: name it "Boost".

You now have a copy of your configuration. Just make the small adjustments needed on this boost profile to make the fans spin faster and you are set. Switching between the boost/default profiles takes less than a second when you right click the tray icon.

**Comment by KoXz666** [comment]
Yea.. This is chaos for me.. After restart, there is last loaded profile applied.. I would be glad to do that "without thinking" Just only when is hot outside and I want to play something.. And not thinking about active profiles etc.. Shame.. :(

**Reply by Rem0o** [reply]
But when it's hot outside, your components will be hotter, so if you configured fan curves properly, they will run a little faster because the temp will be higher.  Goal of fancontrol is to automate everything, no manual operation.

**Reply by KoXz666** [reply]
I dont want to going on higher noise levels all year, when only what is need is boost fans couple times per year. Not sure, what is not understandable or unique :(
